<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>631</width>
     <height>783</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="7">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="MainCam">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
       

       <!-- <item row="0" column="0">
    <widget class="QSlider" name="SliderTime">
        <property name="minimumSize">
            <size>
                <width>16777215</width>
                <height>20</height>
            </size>
        </property>
        <property name="toolTip">
            <string>Video Time</string>
        </property>
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
    </widget>
</item> -->

    <item row="0" column="0" colspan="2">
    <widget class="QSlider" name="SliderTime">
        <property name="minimumSize">
            <size>
                <width>0</width>
                <height>20</height>
            </size>
        </property>
        <property name="toolTip">
            <string>Video Time</string>
        </property>
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
        <property name="maximum">
            <number>100</number>
        </property>
        <property name="pageStep">
            <number>10</number>
        </property>
    </widget>
</item>

        <item row="3" column="1">
         <widget class="QSlider" name="MaxSSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Saturation</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="maxhsv">
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Hue | Saturation | Value</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSlider" name="MinHSlider">
          <property name="toolTip">
           <string>Hue</string>
          </property>
          <property name="maximum">
           <number>179</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="MaxVSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Value</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="MaxHSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Hue</string>
          </property>
          <property name="maximum">
           <number>179</number>
          </property>
          <property name="value">
           <number>179</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QSlider" name="MinVSlider">
          <property name="toolTip">
           <string>Value</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="MinSSlider">
          <property name="toolTip">
           <string>Saturation</string>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="minhsv">
          <property name="maximumSize">
           <size>
            <width>900</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Hue | Saturation | Value</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineFPS"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pushButton</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>757</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>757</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
